gatsby-remark-oembed/.eslintrc

```diff
@@ -1,13 +1,12 @@
 {
   "env": {
+    "es6": true,
     "node": true,
-    "browser": true,
-    "commonjs": true,
     "jest/globals": true
   },
   "plugins": ["jest"],
   "parserOptions": {
-    "ecmaVersion": "2018"
+    "ecmaVersion": 2018
```

Comment:
For the 2018 version to be picked up, it needs to be a number, not a string.

![image](https://user-images.githubusercontent.com/833231/97255683-8b7c5e80-17e7-11eb-92cf-80db36586f2c.png)